cromwell-helm/templates/cromwell.yaml

```diff
@@ -166,6 +166,54 @@ data:
             }
           }
         }
+        GCPBATCH {
+          actor-factory = "cromwell.backend.google.batch.GcpBatchBackendLifecycleActorFactory"
+          config {
+            // Google project
+            project = "{{ .Values.config.gcsProject }}"
+
+            // Base bucket for workflow executions
+            root = "{{ .Values.config.gcsBucket }}"
+
+            // Polling for completion backs-off gradually for slower-running jobs.
+            // This is the maximum polling interval (in seconds):
+            maximum-polling-interval = 600
+
+            // Optional Dockerhub Credentials. Can be used to access private docker images.
+            dockerhub {
+              // account = ""
+              // token = ""
+            }
+
+            batch {
+              // A reference to an auth defined in the `google` stanza at the top.  This auth is used to create
+              // Batch Jobs and manipulate auth JSONs.
+              auth = "application-default"
+
+              // alternative service account to use on the launched compute instance
+              // NOTE: If combined with service account authorization, both that service account and this service account
+              // must be able to read and write to the 'root' GCS path
+              compute-service-account = "{{ .Values.config.serviceAccount.annotations.gcpServiceAccount }}"
+
+              // Location to submit jobs to Batch and store job metadata.
+              location = "{{ .Values.config.gcsRegion }}"
+            }
+
+            filesystems {
+              gcs {
+                // A reference to a potentially different auth for manipulating files via engine functions.
+                auth = "application-default"
+                // Google project which will be billed for the requests
+                project = "{{ .Values.config.gcsProject }}"
+              }
+            }
+            virtual-private-cloud {
+              network-label-key = "vpc-network-name"
+              subnetwork-label-key = "vpc-subnetwork-name"
+              auth = "application-default"
+            }    
```

Comment:
Batch in Terra is [a bit different here](https://github.com/broadinstitute/terra-helmfile/blob/88dc8a48684dafebf6ba2f213a27d398332badee/charts/cromwell/templates/config/_cromwell-gcp-batch.conf.tpl#L43-L47), not sure if it applies to this use case as well.

Reply:
Good catch, thanks! I was basing this on the `GCPBATCH.conf` file in the Cromwell repo, but the actual helm file template config is likely more accurate than the example configs

Reply:
Ah yeah, unfortunately that may be out of date or intended for a generic not-Terra installation.